backend/routes/uploadRoutes.js

```diff
@@ -15,26 +16,33 @@ const storage = multer.diskStorage({
   },
 });
 
-function checkFileType(file, cb) {
```

Comment:
The checkFileType function requires a [req parameter](https://github.com/expressjs/multer#filefilter).